physical_properties/pKa/example_submission_files/typeII-MehtapIsik-1.csv

```diff
@@ -0,0 +1,63 @@
+# Submission Type II

```

Comment:
Maybe we should specify at the top of each file that any line that begins with a `#` is considered a comment and will be ignored when parsing.

Reply:
This is a good idea. I will add this.

Reply:
Done.